spec/velveteen/worker_spec.rb

```diff
@@ -73,11 +87,19 @@ class TestSchemaWorker < described_class
   class TestPublishingWorker < described_class
     attr_accessor :test_headers
 
-    def perform
+    def perform(override_message = nil, publishing_key = "velveteen.test.publish")
```

Comment:
This update introduces some complexity that I'm kinda feeling needs its own tests. What do you think about overriding `initialize` in the class to allow specifying a routing key? Would that be enough to support the tests here?

Reply:
I agree, it is smelly. That's a good idea, let me try to address this.